plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -309,7 +328,9 @@ export class PersonState {
         })
         Object.entries(properties).map(([key, value]) => {
             if (personProperties[key] !== value) {
-                updated = true
+                if (this.shouldUpdatePersonIfOnlyChange(key)) {
```

Comment:
Should we check if the existing key is `undefined` (like above) and set `updated = true` if so? So that at least the first time a property is seen it is saved? Or no?

Reply:
ah yes good idea I'll add that

Reply:
I actually don't know if it would be more confusing or not. It would mean the user profile would forever show the first `$current_url` I guess?

Reply:
if there's any other updates it will do an update too, it just won't do an update if it's the only change, so that's probably a rare use case